utils.py

```diff
@@ -42,7 +42,14 @@ def n(v):
         return 0L
     if v.endswith('B'):
         return long(v[:-1])
-    return long(float(v))
+    return dbus.UInt64(long(float(v)))
+
+def n32(v):
```

Comment:
Sorry, that was ambiguous. I was wondering why cast to ```long```, when Uint32 is short, relatively speaking. Why not cast to ```int``` and so forth, in *32 method?